@noun-auction/components/DataRenderers/AuctionHighBid.tsx

```diff
@@ -1,50 +1,63 @@
-import { SharedDataRendererProps, useNounishAuctionProvider } from '@noun-auction'
+import { useMemo } from 'react'
+
 import { sidebarHighBid } from '@noun-auction/styles/NounishStyles.css'
-// @shared
 import { lightFont } from '@shared'
 import { Flex, Label } from '@zoralabs/zord'
 
 import { EthAmount } from './EthAmount'
 
+type Props = {
+  showLabels?: boolean
+  layout: string
+  highestBid?: string
+  collectionAddress: string
+  className?: any
+  auctionCompleted: boolean
+  styles: {
+    [k: string]: any
+  }
+}
+
 export function AuctionHighBid({
-  label = 'Current bid',
-  layoutDirection = 'row',
   showLabels,
-  ...props
-}: {
-  useUsdc?: boolean
-} & SharedDataRendererProps) {
-  const { layout, timerComplete, activeAuction } = useNounishAuctionProvider()
-
-  const highestBid = activeAuction?.properties?.highestBidPrice?.chainTokenPrice?.raw
+  highestBid,
+  layout,
+  styles,
+  auctionCompleted,
+}: Props) {
+  const label = styles.label || 'Current bid'
+  const layoutDirection = styles.layoutDirection || 'row'
```

Comment:
Likewise here, move layout="row" as default prop and extend FlexProps to use CSS properties